aws-provider/postgresdb/README.md

```diff
@@ -0,0 +1,172 @@
+# `crossplane` PostgresDB example
+
+![RDS layout](./crossplane-rds.drawio.png)
+
+In contrast to the other examples in this [AWS provider series](../), this more
+complex example will build a full network hosting an Amazon Postgresql RDS
+database, peered to the cluster VPC with routes between each.
+
+The example contains three subnets, one for each availability zone, leaving
+space for read replicas to be created in zones B and C although the example
+does not actually implement these replicas.
+
+## Initial setup
+
+Before this example can be built, you must first enable crossplane AWS provider
+family members and create the primary role for crossplane to assume.
+
+In this example, we need the following provider family members enabled inside the
+cluster.
+
+- [provider-aws-ec2](https://marketplace.upbound.io/providers/upbound/provider-aws-ec2/)
+- [provider-aws-rds](https://marketplace.upbound.io/providers/upbound/provider-aws-rds/)
+- [provider-aws-kms](https://marketplace.upbound.io/providers/upbound/provider-aws-kms/)
+
+To enable these, please follow the instructions in [management-cluster-bases/extras/crossplane/providers/upbound/aws](https://github.com/giantswarm/management-cluster-bases/tree/main/extras/crossplane/providers/upbound/aws)
+including the instruction on [Using crossplane with IAM Roles for Service Accounts](https://github.com/giantswarm/management-cluster-bases/tree/main/extras/crossplane/providers/upbound/aws#using-crossplane-with-iam-roles-for-service-accounts)
+
+## Create the RDS role
+
+Once the providers have been enabled, and the primary assume role has been
+created, we now need to create a second role inside the account for crossplane
+to use with permissions to create and manage components inside the account.
+
+For a detailed explanation of which permissions are required and why, please
+see [this linked policy document](./policies/README.md).
+
+### Step by step installation guide
+
+1. Edit the file [`rds-crossplane-policy](./policies/rds-crossplane-policy.json)
+   and set the variable `${AWS_ACCOUNT_ID}` to the ID of the AWS account you're
+   going to use.
+
+   ```bash
+   export POLICY_NAME="rds-crossplane-policy"
+   export AWS_ACCOUNT_ID=$(aws sts get-caller-identity | jq -rn .Account)
+
+   cat ./policies/${POLICY_NAME}.json | envsubst \
+       | sponge ./policies/${POLICY_NAME}.json
```

Comment:
Why not just redirect the output to the file? I dont really know `sponge`, just quickly giving it a try it is equivalent to:

```bash
cat ./policies/${POLICY_NAME}.json | envsubst > ./policies/${POLICY_NAME}.json
```

Just so that it is one less tool needed to follow the example. `envsubst` is not on mac either by default, but needed.

Reply:
I did this for some level of consitency between the handling of outputs of tooling. 

`jq` does not allow for `--inplace` or `-i` functionality in the same way that `yq` and `sed` have which, if means `jq file.json > file.json` actually forces truncation of the file.

Whilst a better argument here would be "Why not use `yq` for handling JSON", I feel this drives some level of confusion when using a tool designed for yaml against json policies.

To maintain a sense of similarity between the command outputs I re-used the pipe to `sponge` even though this is a redundant requirement.

Regarding `envsubst` this is called out already on the [`README.md`](https://github.com/giantswarm/crossplane-examples/blob/main/README.md) for the repository as a requirement.